server/db/migration/migrations/sqls/20200129121417-extent-of-forest-view-up.sql

```diff
@@ -1,32 +1,62 @@
-DROP VIEW IF EXISTS public.extent_of_forest_view;
-CREATE VIEW
-  public.extent_of_forest_view AS
-  (
-    SELECT
-      f.country_iso,
-      f.year,
-      f.forest_area,
-      f.other_wooded_land
-    FROM
-      public.eof_fra_values f
-    WHERE
-      f.year NOT IN
-      (
-        SELECT
-          n.year
-        FROM
-          public.ndp_view n
-        WHERE
-          n.country_iso = f.country_iso )
-    UNION ALL
-    SELECT
-      n.country_iso,
-      n.year,
-      n.forest_area,
-      n.other_wooded_land_area AS other_wooded_land
-    FROM
-      public.ndp_view n
-    ORDER BY
-      1,2
-  );
-  
\ No newline at end of file
+DROP VIEW IF EXISTS extent_of_forest_view;
+CREATE VIEW extent_of_forest_view AS
+(
+WITH extent_of_forest AS (
+    SELECT *, row_number() OVER (PARTITION BY country_iso, year) AS row_number
+    FROM (
+        SELECT n.country_iso,
+               n.year,
+               1                        AS source,
+               n.forest_area,
+               n.other_wooded_land_area AS other_wooded_land
+        FROM ndp_view n
+        UNION
+        SELECT f.country_iso,
+               f.year,
+               2 AS source,
+               f.forest_area,
+               f.other_wooded_land
+        FROM eof_fra_values f
+        ORDER BY 1, 2, 3
+    ) AS q
+),
+     q2 AS (
+         SELECT e.country_iso,
+                'other_wooded_land'                                   AS row_name,
+                SUM(e.other_wooded_land) FILTER (WHERE e.year = 1990) AS "1990",
+                SUM(e.other_wooded_land) FILTER (WHERE e.year = 2000) AS "2000",
+                SUM(e.other_wooded_land) FILTER (WHERE e.year = 2010) AS "2010",
+                SUM(e.other_wooded_land) FILTER (WHERE e.year = 2015) AS "2015",
+                SUM(e.other_wooded_land) FILTER (WHERE e.year = 2020) AS "2020"
+         FROM extent_of_forest e
+         WHERE e.row_number = 1
+         GROUP BY e.country_iso
+         UNION
+         SELECT e.country_iso,
+                'forest_area'                                   AS row_name,
+                SUM(e.forest_area) FILTER (WHERE e.year = 1990) AS "1990",
+                SUM(e.forest_area) FILTER (WHERE e.year = 2000) AS "2000",
+                SUM(e.forest_area) FILTER (WHERE e.year = 2010) AS "2010",
+                SUM(e.forest_area) FILTER (WHERE e.year = 2015) AS "2015",
+                SUM(e.forest_area) FILTER (WHERE e.year = 2020) AS "2020"
+         FROM extent_of_forest e
+         WHERE e.row_number = 1
+         GROUP BY e.country_iso
+         ORDER BY 1, 2
+     )
+SELECT *
+FROM q2
+UNION
+SELECT q2.country_iso,
+       'forest_area_percent'                                                                 AS row_name,
```

Comment:
do not add forest_percent here. but rather total_land_area as it's needed for data export module. 
forest_percent can be added to statistical_factsheets_view